dali/server/daserver.cpp

```diff
@@ -255,12 +255,22 @@ static bool populateWhiteListFromEnvironment(IWhiteListWriter &writer)
                 {
                     const char *masterCompName = component.queryProp("ThorMasterProcess/@computer");
                     StringBuffer ipSB;
-                    const char *ip = resolveComputer(masterCompName, component.queryProp("@netAddress"), ipSB);
+                    const char *ip = resolveComputer(masterCompName, nullptr, ipSB);
```

Comment:
This doesn't look like it is related.  What was the reason for this change?

Reply:
no not really related, but spotted that that parameter isn't used - the topology of a ThorMasterProcess/ThorSlaveProcess, does not have a @netAddress, it is always looked up via the computer name.